chapter08/red/exercise2.pl

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+my %handles;
+while (<>) {
+  my ( $castaway ) = split /:/;
+
+  if ( !defined $handles{$castaway} ) {
+    open $handles{$castaway}, '>', lc $castaway . '.log'
```

Comment:
Might have been a bit safer to sanitise the name here - names could potentially contain characters which don't play nicely with the Prof's frail old coconut-powered computer.